src/main/java/logisticspipes/renderer/LogisticsRenderPipe.java

```diff
@@ -503,10 +502,10 @@ private void renderItem(IIcon par3Icon) {
         Tessellator tessellator = Tessellator.instance;
         tessellator.startDrawingQuads();
         tessellator.setNormal(0.0F, 1.0F, 0.0F);
-        tessellator.addVertexWithUV(par1 + 0, par2 + par5, zLevel, par3Icon.getMinU(), par3Icon.getMaxV());
```

Comment:
Zeros need to be left to improve readability of Vertex adding (the same for lines below)